### DISCUSSION TITLE
Any plans on when the inlineCompletionsAdditions proposed API could be finalized?

### DISCUSSION THREAD

**Discussion by gbockus-sf** [microsoft/vscode-discussions#1261] [Extension Development QnA]
Howdy,
I work on an extension that utilizes the InlineCompletionsProvider.  It'd be awesome if the methods for `InlineCompletionItemProvider` found in https://github.com/microsoft/vscode/blob/main/src/vscode-dts/vscode.proposed.inlineCompletionsAdditions.d.ts were finalized and shipped so they could be used by extension in the marketplace. 

`handleDidShowCompletionItem` would be especially useful for knowing if the end user saw the supplied completion. 

I did some looking around but didn't find any info about when these kind of proposed APIs get promoted.  

Thanks!

**Comment by gbockus-sf** [comment]
Howdy just following up on this question.  Also if it belongs elsewhere please let me know.

**Comment by olguzzar** [comment]
@hediet is there any insight you can provide here?

**Comment by gbockus-sf** [comment]
Same Q @hediet. Noticed you updated the spec last month.  Wondering if you have any insight.

**Reply by burkeholland** [reply]
Looking into this

**Comment by burkeholland** [comment]
Closing as it looks like it was promoted: https://github.com/microsoft/vscode-discussions/discussions/26
